manifests/server/puppetserver.pp

```diff
@@ -227,6 +228,17 @@
     }
   }
 
+  file { "${server_puppetserver_dir}/logback.xml":
+    ensure  => file,
+    content => template('puppet/server/puppetserver/logback.xml.erb'),
+  }
+
+  if versioncmp($server_puppetserver_version, '5.3.6') >= 0 {
+    $ca_conf_ensure = present
+  } else {
+    $ca_conf_ensure = absent
+  }
```

Comment:
I guess the rebase/merge went wrong here...

Reply:
Could you have a look here? This still needs to be removed